src/__tests__/ceramic_integration.test.ts

```diff
@@ -1,684 +0,0 @@
-import 'reflect-metadata'
```

Comment:
This test has been flaky for a while but is failing consistently on this PR.  I'm pretty sure it's because I changed the timestamp to be required instead of optional, and we are still using a very old version of js-ceramic in the CAS repo.  Upgrading the CAS to the newest version of js-ceramic is non-trivial and frankly doesn't seem worth it just for test code, now that the CAS doesn't use ceramic in its actual production code.

When I first wrote this test we had very little in the way of test coverage that the CAS was working and that ceramic and the CAS were playing well together.  Nowadays we have a lot more test coverage and have the ceramic-cas integration tested in several other places.  Because of that I think maintaining this test is becoming more trouble than it's worth.

FYI @smrz2001